### ISSUE TITLE
Google Tag Manager is not working?

### ISSUE THREAD

**Issue by paolantoni** [WorldBank-Transport/DRIVER#789]
I noticed that the service is not working well.
The web site of this project https://roadsafety.io/#!/ is reporting https://www.googletagmanager.com/gtm.js?id=GTM-MH8LWD

The service is not working  also if you provide a new walid GTM ID.
What's wrong?

**Comment by paolantoni** [comment]
i suppose the url parameters are wrong.
https://www.googletagmanager.com/**gtag/js**?id=<put your gtag here>
should be correct.

**Comment by rmartz** [comment]
The `gtm.js?id=<ID>` URL matches what is suggested in the [Google Tag Manager Quickstart Guide](https://developers.google.com/tag-manager/quickstart) code snippet, so I believe that should be working as intended. What error are you seeing? Is `google_analytics_id` group var set to the correct ID for your account?

**Comment by paolantoni** [comment]
Yes, the google_analytics_id in my project has been set before proviosioning (with the id i have in my analytics accont), but please have a look to my suggestion because also the **official site** has the same issue (as mentioned).
If you check the page
  https://roadsafety.io/#!/
you'll see (F12):

GET https://www.googletagmanager.com/gtm.js?id=GTM-MH8LWD net::ERR_ABORTED 404
(anonymous) @ (index):1
(anonymous) @ (index):1

**Comment by paolantoni** [comment]
Updated to 2.0.5 but same error:
`www.googletagmanager.com/gtm.js?id=UA-141209618-1:1 Failed to load resource: the server responded with a status of 404 ()`

**Comment by imagebind** [comment]
Hi @paolantoni 
Do you have versions in GTM?  you need to publish it if there are no versions.
Please read [this](https://karpi.xyz/tag-managers/gtm/gtm-404-error/) doc for more steps to solve it.
